2020/3/trees/tests/tests.rs

```diff
@@ -0,0 +1,71 @@
+use trees::*;
+
+#[test]
```

Comment:
Tests! Nice!

Generally, it's idiomatic to put unit tests in the same file as the code. They'll be compiled out of the non-test binary. If you need support functions, etc. and don't want those compiled into the release binary you can mark them `#[cfg(test)]` and they'll only be included in tests (`cfg` ==> `ConditionalAttribute` ;))

Reply:
Will do from now on.